src/bundle/Resources/config/universal_discovery_widget.yaml

```
@@ -76,3 +76,5 @@ system:
                     multiple: false
                 subtree_search:
                     multiple: false
+                image_picker:
+                    activeTab: 'browse'
```

Maybe:
```suggestion
                    active_tab: 'browse'
```
to follow the convention?

---

Good point!